en/config/actuators.md

```diff
@@ -0,0 +1,140 @@
+:::
+
+
+### Motor Tilt Servos
+
+Tilt servos are configured as follows:
+- The reference direction is upwards (negative Z direction).
+- Tilt direction: **Towards Front** means the servo tilts towards positive X direction, whereas **Towards Right** means towards positive Y direction.
+- Minimum and maximum tilt angles: specify the physical limits in degrees of the tilt at minimum control and maximum respectively.
+  An angle of 0° means to point upwards, then increases towards the tilt direction.
+  :::note
+  Negative angles are possible. For example tiltable multirotors have symmetrical limits and one could specify -30 as minimum and 30 degrees as maximum.
+  :::
+  :::note
+  If a motor/tilt points downwards and tilts towards the back it is logically equivalent to a motor pointing upwards and tilting towards the front.
+  :::
+- Control: depending on the airframe, tilt servos can be used to control torque on one or more axis (it's possible to only use a subset of the available tilts for a certain torque control):
+  - Yaw: the tilts are used to control yaw (generally desired).
+    If four or more motors are used, the motors can be used instead.
+  - Pitch: typically differential motor thrust is used to control pitch, but some airframes require pitch to be controlled by the tilt servos.
+    Bicopters are among those.
+- Tiltable motors are then assigned to one of the tilt servos.
+
+![Tilt Axis](../../assets/config/actuators/tilt_axis.png)
+
+
+### Reversible Motors
+
+If reversible motor(s) are used (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half), make sure to select the **Reversible** checkbox for those motor(s).
+Also ensure the ESC is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
+
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+
+
+## Actuator Outputs
+
+The actuators and any other output function can be assigned to any of the physical outputs.
+Each output has its own tab, e.g. the PWM MAIN or AUX output pins, or UAVCAN.
+
+PWM outputs are grouped according to the hardware groups of the autopilot.
+Each group allows to configure the PWM rate or DShot/Oneshot (if supported).
+
+:::note
+For boards with MAIN and AUX, prefer the AUX pins over the MAIN pins for motors, as they have lower latency.
+:::
+
+The AUX pins have additional configuration options for camera capture/triggering.
+Selecting these requires a reboot before they are applied.
+
+
+## Actuator Testing
+
+When testing actuators, make sure that:
+- Motors spin at the "minimum thrust" position.
+
+  The sliders snap into place at the lower end, and motors are turned off (disarmed).
+  The "minimum thrust" position is the next slider position, which commands the minimum thrust.
+  Adjust the minimum output value such that the motors spin at that slider position.
```

Comment:
How do we do this? On this screen we can configure max/min value for some servos but not for motors.

Reply:
We can mention that this is mostly relevant when PWM is used. For DShot it's not required.

Reply:
I can add that, but can you confirm that, for PWM, this is something you have to do outside of this screen/configuration? i wonder if we should provide PWM range setting controls for motors using PWM?

Reply:
No, min/max will show up if you have PWM selected (as in the screenshot), and only be hidden if DShot is selected.